src/Models/projects.js

```diff
@@ -34,7 +34,7 @@ const projectSchema = new mongoose.Schema({
       client: {
         type: mg.Types.ObjectId, ref: 'clients'
       },
-      office: {
+      offices: {
```

Comment:
@KristomeisterDev lo mismo debe ser en singular.
```suggestion
      office: {
```

Reply:
asi es, se cambio en varios, ahora los corrijo